src/widgettoolbarrepository.js

```diff
@@ -117,7 +117,9 @@ export default class WidgetToolbarRepository extends Plugin {
 	 */
 	register( toolbarId, { items, getRelatedElement, balloonClassName = 'ck-toolbar-container' } ) {
 		const editor = this.editor;
-		const toolbarView = new ToolbarView();
+		const locale = this.editor.locale;
+		const t = locale.t;
+		const toolbarView = new ToolbarView( locale, { ariaLabel: t( 'Widget\'s toolbar' ) } );
```

Comment:
1. As I pointed out in https://github.com/ckeditor/ckeditor5-ui/pull/500, the `ariaLabel` should be a property.
2. `#register` should allow customization of `ariaLabel` (default `t( 'Widget toolbar' )`).
3. Having the above, we need PRs in 
   * ckeditor5-image
   * ckeditor5-table
   * ckeditor5-media-embed

   that will correctly describe widgets introduced by the features ("Image toolbar", "Table toolbar", etc.)

Reply:
I've updated the description for PR to link changes in image, table, media-embed.